test/specs/Conformance-test.js

```diff
@@ -102,44 +83,9 @@ describe('Conformance', () => {
             })
             .should.equal(true)
         })
-        describe('className', () => {
-          it(`has props.className after "${sdClass}"`, () => {
-            let renderedClasses
-            const rendered = render(<SDComponent className={classes} />)
-              .findClass(sdClass)
-            if (isDOMComponent(rendered)) {
-              renderedClasses = rendered.getAttribute('class')
-            } else {
-              renderedClasses = rendered.props.className
-            }
-            const sdClassIndex = renderedClasses.indexOf(sdClass)
-            const classesIndex = renderedClasses.indexOf(classes)
-            expect(sdClassIndex).to.be.below(classesIndex)
-          })
```

Comment:
No longer check order, with nested private components it is not reasonably feasible.  We still test that the classes are there, just not that the first child has them.

The `_Header` case was the deciding factor.  The first child is the private header.  The actual generated `sd-*` class is not the first child.  We could _assume_ that the second child will have the class, but that only works right now in this one case.  If we use 2 private wrappers for something else, it will break (`sd-*` would be child 3).

We dropped the ordering requirement for now after considering the other options.